addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -346,6 +346,9 @@ var SnippetEditor = Widget.extend({
 
         // Show/hide overlay in preview mode or not
         this.$el.toggleClass('oe_active', show);
+        const $optionWrap = this.$el.find('.o_overlay_options_wrap');
```

Comment:
Can we avoid to search the DOM for this each time the overlay is shown/hidden (especially since not needed when hidden) ?